src/browserClip.ts

```diff
@@ -215,10 +217,12 @@ const generateSequence = (params: ClipParams) => {
     if (params.volume) {
       params.sampler.volume.value = params.volume;
     }
-    params.sampler.chain(
-      ...effects,
-      Tone.Master ? Tone.Master : Tone.Destination
-    );
+    params.sampler.chain(...effects);
+    if (isToneV13()) {
```

Comment:
Apologies for bringing this up a bit later than it s already merged but are earlier versions of Tone capable of whatever s in the else?

Reply:
I don't understand this sentence 😅. Can you be more specific ? 😬

Tone version 13 is able to do dynamic rendering (all the scribbletune features except offline rendering), but I guess you know that.

Note: the lines above are only to harmonize the way we handle tone versions in the code. So that when tone 14 is published officially, it will be easier to clean the code, finding all the `if else` statements using `isToneV13()` as condition.